### ISSUE TITLE
lxd can only be ^C'd until a container has started

### ISSUE THREAD

**Issue by tych0** [lxc/lxd#122]
After a container starts (even if that container is stopped), lxd can't be ^C'd. We should probably either not allow it at all, or allow it always and kill all the containers if it is ^C'd. I prefer the second option.

**Comment by hallyn** [comment]
Hi, just to be clear, are you running with ppa:ubuntu-lxc/daily?  Because the ctrl-c bug should have been fixed not too long ago, when I originally found it while working on the flex attach method.

**Comment by tych0** [comment]
On Tue, Jan 06, 2015 at 08:12:02AM -0800, hallyn wrote:
> Hi, just to be clear, are you running with ppa:ubuntu-lxc/daily?  Because the ctrl-c bug should have been fixed not too long ago, when I originally found it while working on the flex attach method.

I've seen this with the trunk at least as of last month when I paused
lxd development. ^C works in some cases, but not others (but whether
or not it worked didn't have anything to do with attach, I didn't
think). Unfortunately I just fiddled around with it and I can't seem
to reproduce it. We can close this for now and I can re-open when I
see the behavior again with steps to reproduce.
